src/proposal.mligo

```diff
@@ -316,18 +328,21 @@ let is_voting_period_over (proposal, voting_period, store : proposal * voting_pe
   current_period > proposal.period_num + 1n
 
 [@inline]
-let do_total_vote_meet_quorum_threshold (proposal, store, quorum_threshold : proposal * storage * quorum_threshold): bool =
+let frozen_total_supply(store : storage): nat =
+    match Map.find_opt store.frozen_token_id store.total_supply with
+    | Some v -> v
+    | None -> ((failwith "BAD_STATE") : nat)
+
+[@inline]
+let do_total_vote_meet_quorum_threshold (proposal, store, voting_period, quorum_threshold : proposal * storage * voting_period * quorum_threshold): bool =
```

Comment:
`voting_period` was added here, but it's not used.

Reply:
Not sure why it was added, but have removed it now.